src/main/Port.Adapter/In/Api/Settings/EnvironmentSettingsService.cs

```diff
@@ -1,14 +1,24 @@
-﻿using ei8.Cortex.Subscriptions.Application.Interface.Service;
+﻿using ei8.Cortex.Subscriptions.Domain.Model;
+using ei8.Net.Http.Notifications;
 
 namespace ei8.Cortex.Subscriptions.In.Api.Settings
 {
     public class EnvironmentSettingsService : ISettingsService
     {
         public EnvironmentSettingsService(IConfiguration config)
         {
-            SubscriptionsDatabasePath = config.GetSection(EnvironmentVariableKeys.SubscriptionsDatabasePath).Value;
+            DatabasePath = config.GetValue<string>(EnvironmentVariableKeys.SubscriptionsDatabasePath);
+            PollingIntervalSeconds = config.GetValue<int>(EnvironmentVariableKeys.SubscriptionsPollingIntervalSeconds);
```

Comment:
Please add "this." to instance references.

Reply:
@elmerbool This is done